R/pkg/tests/fulltests/test_mllib_classification.R

```diff
@@ -313,7 +313,7 @@ test_that("spark.mlp", {
   # Test predict method
```

Comment:
Actually I think we may remove this test `Test predict method`. Seems to me, with the `tol = 0.5`, the prediction may not be very meaningful.